src/containers/quiz/quiz-header/QuizHeader.tsx

```diff
@@ -1,16 +1,22 @@
 import { useAppSelector } from '~/hooks/use-redux'
 import Box from '@mui/material/Box'
 
-import { UserRoleEnum } from '~/types'
+import { Question, QuizAttempt, QuizTimeLimit, UserRoleEnum } from '~/types'
 import {
   ActiveQuizInfo,
   FinishedQuizInfo,
   UngradedQuizInfo,
-  GradedQuizInfo
+  GradedQuizInfo,
+  StartViewQuizInfo
 } from '~/containers/quiz/quiz-info/QuizInfo'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 
 import styles from '~/containers/quiz/quiz-header/QuizHeader.styles'
+import { useQuery } from '@tanstack/react-query'
```

Comment:
```suggestion
import { useQuery } from 'path-to-custom-hook'
```
You can use our custom wrapper instead.

Reply:
oh, auto import didn't work correct. I thought I used our custom hook